content/reference/gql-types.md

```diff
@@ -0,0 +1,122 @@
+---
+title: GraphQL types and filters
+
+description: >-
+  A reference of the data types in the Rhize API and of the filters available for each type.
+categories: ["reference"]
+weight: 930
+menu:
+  main:
+    name: GQL types and filters
+    identifier: gql-filters
+    parent: reference
+---
+
+This page provides a reference of the data types enforced by the Rhize database schema,
+and of the filters that can apply to these types when you query, update, or delete a set of resources.
+For an extended guide, with examples, read [Use query filters]({{< relref "/how-to/gql/filter" >}}).
+
+{{< notice "note" >}}
+These filters are based on Rhize's implementation of the Dgraph [`@search` directives](https://dgraph.io/docs/graphql/schema/directives/search/).
+{{< /notice >}}
+
+## Data types
+
+Every object in the Rhize schema has fields that are of one of the basic data types.
+From the other perspective, these data types define fields that compose manufacturing objects,
+objects defined precisely by ISA-95 and enforced by Rhize's database schema.
+
+### Basic types
+
+Every manufacturing object in the Rhize database is made of fields that are of one the following basic types.
+In official GraphQL terminology, these types are called [_scalar types_](https://graphql.org/learn/schema/#scalar-types).
+
+- `String`
+- `Int`
+- `Float`
+- `Enum`
+- `Boolean`
```

Comment:
Should we define these? I could see most people getting string and maybe int, but float and enum especially are not terms most people are familiar with, though not sure about people from a manufacturing background.